src/middlewared/middlewared/etc_files/generate_ssl_certs.py

```diff
@@ -26,9 +26,8 @@ def write_certificates(certs: list, cacerts: list) -> set:
             with open(cert['csr_path'], 'w') as f:
                 f.write(cert['CSR'])
 
-    trusted_cas_path = '/usr/local/share/ca-certificates'
+    trusted_cas_path = '/var/local/ca-certificates'
     shutil.rmtree(trusted_cas_path, ignore_errors=True)
-    os.makedirs(trusted_cas_path)
```

Comment:
Why? How will `f.write` call below work if we just removed the directory above?

Reply:
It's a dataset.

Reply:
Please leave a comment then, it's not obvious what is happening here

Reply:
Okay. Done.